src/pages/venues/[id]/map/index.tsx

```diff
@@ -0,0 +1,144 @@
+import { ApolloProvider, isApolloError, useQuery } from "@apollo/client";
+import { IconArrowLeft } from "hds-react";
+import { GetStaticPropsContext } from "next";
+import dynamic from "next/dynamic";
+import React from "react";
+
+import { VENUE_QUERY } from "..";
+import Page from "../../../../common/components/page/Page";
+import initializeCmsApollo from "../../../../domain/clients/cmsApolloClient";
+import initializeNextApiApolloClient, {
+  useNextApiApolloClient,
+} from "../../../../domain/clients/nextApiApolloClient";
+import Link from "../../../../domain/i18n/router/Link";
+import useRouter from "../../../../domain/i18n/router/useRouter";
+import { getLocaleOrError } from "../../../../domain/i18n/router/utils";
+import serverSideTranslationsWithCommon from "../../../../domain/i18n/serverSideTranslationsWithCommon";
+import { staticGenerationLogger } from "../../../../domain/logger";
+import styles from "./map.module.scss";
+
+const MapView = dynamic(
+  () => import("../../../../common/components/mapView/MapView"),
+  {
+    ssr: false,
+  }
+);
+
+const VenueMapPageContent: React.FC = () => {
+  const router = useRouter();
+  const { data, loading, error } = useQuery(VENUE_QUERY, {
+    variables: {
+      id: router.query.id,
+    },
+  });
+
+  if (loading || error) {
```

Comment:
I'm wondering about this 🤔 In essence it shouldn't really matter in this case because (1) the data should be available when we first render the map in the client and (b) refocusing the map could be awkward UX. But generally speaking, I think, that we should render the map as early as possible so that the images etc. have as much time as possible to load.